src/FSharp.Control.AsyncSeq/AsyncSeq.fs

```diff
@@ -1483,6 +1483,21 @@ module AsyncSeq =
       if (buffer.Count > 0) then
           yield buffer.ToArray() }
 
+  let toSortedSeq fn source =
+    toArrayAsync source |> Async.map (fun source' -> (fn source' :> seq<'T>)) |> Async.RunSynchronously
```

Comment:
Hmmmm.... since they all use `toArrayAsync` maybe the return type should just be an honest array!

Reply:
Yes, done 👍